ffs/filcold/filcold.go

```diff
@@ -69,10 +74,14 @@ func (fc *FilCold) Store(ctx context.Context, c cid.Cid, cfg ffs.FilConfig) ([]c
 	if err != nil {
 		return nil, nil, 0, fmt.Errorf("making deal configs: %s", err)
 	}
-	size, err := fc.getCidSize(ctx, c)
+
+	fc.l.Log(ctx, c, "Calculating piece size...")
+	size, err := fc.calculatePieceSize(ctx, c)
 	if err != nil {
 		return nil, nil, 0, fmt.Errorf("getting cid cummulative size: %s", err)
 	}
+	fc.l.Log(ctx, c, "Estimated piece size is %d bytes.", size)
```

Comment:
The piece size calculation is added to the human-friendly logs because it isn't that much negligible work.